internal/ansible/events/log_events.go

```diff
@@ -62,22 +64,50 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 		"job", ident,
 	)
 
+	// Parse verbosity from CR
+	verbosityAnnotation := 0
```

Comment:
I think it might be better to move all of these verbosity parsing blocks to a separate method, so we can just call `verbosity := getVerbosity(u)`, might make it a little more clear